test/notificationSpec.js

```diff
@@ -844,92 +913,92 @@ describe('Notifications', function () {
         .expect(200)
         .end()
     })
-  })
-
-  describe('atomic mode: one case', function () {
-    beforeEach(function () {
-      this.source_transfer = this.notificationSourceTransferAtomic.resource
-      this.destination_transfer = this.source_transfer.credits[0].memo.destination_transfer
-    })
-
-    it('should check the expiry on a cancellation condition: too long', function * () {
-      const caseID = this.destination_transfer.additional_info.cases[0]
-      nock(caseID)
-        .get('')
-        .reply(200, { expires_at: future(15000) })
-
-      yield this.request()
-        .post('/notifications')
-        .send(this.notificationSourceTransferAtomic)
-        .expect(200)
-        .end()
-    })
-
-    it('should check the expiry on a cancellation condition: already expired', function * () {
-      const caseID = this.destination_transfer.additional_info.cases[0]
-      nock(caseID)
-        .get('')
-        .reply(200, { expires_at: future(-15000) })
-
-      yield this.request()
-        .post('/notifications')
-        .send(this.notificationSourceTransferAtomic)
-        .expect(200)
-        .end()
-    })
 
-    it('should check the expiry on a cancellation condition: missing expiry', function * () {
-      const caseID = this.destination_transfer.additional_info.cases[0]
-      nock(caseID)
-        .get('')
-        .reply(200, {})
-
-      yield this.request()
-        .post('/notifications')
-        .send(this.notificationSourceTransferAtomic)
-        .expect(200)
-        .end()
-    })
-  })
+    describe('atomic mode: one case', function () {
```

Comment:
It looks like the test changes below here should be in a different PR (or the PR description should be updated to reflect these changes)